### DISCUSSION TITLE
Electronic Buzzing Noise

### DISCUSSION THREAD

**Discussion by ericlathrop** [ElectroTechnique/electrotechnique.github.io#72] [Hardware Issues]
I'm getting an electronic buzzing noise on the headphones out. It sounds kinda like I'm hearing the CPU of the synth or something. It doesn't sound like a ground loop. The volume of the noise does not change when I change the volume knob. I soldered a cable to the line out on the audio shield, and the noise is there, too, but everything is a lower volume. How can I remove the noise?

**Comment by ericlathrop** [comment]
It was a ground loop after all. When I switched to using a phone charger to power the TSynth, and a normal MIDI cable, the buzzing went away. Is there a way to mod the circuit to have a separate power supply so I can use USB for data without a ground loop?

**Reply by Electrotechnique** [reply]
Sorry for the long wait for a reply. These cheap USB isolators work fine, although only support 'normal' speed USB (12Mbps) I think, but are fine with the synth.

[USB Isolator](https://www.aliexpress.com/item/1005002946734805.html?spm=a2g0o.productlist.0.0.68db16edrdji6u&algo_pvid=b917cde8-fa99-4270-95f6-4bf8db9befde&aem_p4p_detail=202110161816214509197338752400010530526&algo_exp_id=b917cde8-fa99-4270-95f6-4bf8db9befde-9&pdp_ext_f=%7B%22sku_id%22%3A%2212000022911257870%22%7D)
